ipwb/replay.py

```diff
@@ -575,12 +575,14 @@ def showAdmin():
                'htmlCount': htmlCount,
                'earliest': oldest_datetime,
                'latest': newest_datetime}
+
+    # TODO: determine why summary.htmlCount work propagate to template
     return render_template('admin.html', status=status, indexes=indexes,
-                           summary=summary)
+                           uris=uris, htmlCount=htmlCount, summary=summary)
```

Comment:
Why are we supplying `uris` and `htmlCount` separately when they are already part of the `summary` object?

Reply:
@ibnesayeed See the comment just preceding 578. There is an issue of the values of summary, namely htmlCount, not being available in the template. This is a separate issue. For now, let's at least get the values displayed then refactor the implementation. :-)

Reply:
Resolved in 34882fe42bcd20d8a7b1292d9395fe6524559abd.